src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -336,28 +290,76 @@ class BaseCourseCard extends Component {
     return null;
   };
 
-  renderCourseStartDate = () => {
-    const { startDate, mode } = this.props;
+  renderOrganizationName = () => {
+    const { orgName, mode } = this.props;
+
     const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
+    const courseTypeLabel = isExecutiveEducation2UCourse ? 'Executive Education' : 'Course';
+    const tooltipText = isExecutiveEducation2UCourse
+      ? 'Executive Education courses are instructor-led, cohort-based, and follow a set schedule.'
+      : 'Courses are on-demand, self-paced, and include asynchronous online discussion.';
+
+    if (orgName) {
+      return (
+        <p className={classNames('mb-2 font-weight-light d-flex align-items-center', { 'text-light-300': isExecutiveEducation2UCourse })}>
+          {orgName} &bull; {courseTypeLabel}
+          <OverlayTrigger
+            trigger={['hover', 'focus']}
+            placement="top"
+            overlay={(
+              <Tooltip variant="light" id="tooltip-top">
```

Comment:
nit: might recommend generating a unique tooltip `id` in case multiple `Tooltips` are rendered on the page at once (i.e., IDs need to be unique to be valid HTML).